index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Cereal</title>
+	<link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+	<article class="holec">
+		<header> 
+			<h1>Honey Bunches of Oats with Almonds</h1>
+		</header>
+		<sectiona>
```

Comment:
HTML5 allows you to use custom elements, so it's not an error, but don't do this. Simply use `<section>`